README.adoc

```diff
@@ -23,16 +23,52 @@ Sometimes we need to hardcode some values in SQL, for that we can use syntax lik
 
 _*In case you want to try this out you can use existing SQL and DDLs, no need to make any changes anywhere, just run the program MicroserviceGenerator.java and generate Spring boot microservice, this is explained in next section. To test the generated Rest API you need database which can be setup from instruction given at .dev/README.md*_
 
+=== Building the application from source
+
+Run the following command to generate a runnable jar:
+
+[source,shell]
+----
+mvn package
+----
+
+A jar named `MSG-1.0-SNAPSHOT.jar` will be generated in the `target` directory.
+
 === Generating the microservice application:
-. Run MicroServiceGenerator.java to generate spring boot application. Application will be generated in the user home directory.
 
-        user home for 3 Major OS:
-          - Windows: C:\Users\[username]
-          - Linux: /home/[username]
-          - Mac: /Users/[username]
+There are two ways you can run MSG:
+
+. From sources with Maven:
+
++
+[source,shell]
+----
+mvn spring-boot:run -Dspring-boot.run.arguments="--destination /tmp/test"
```

Comment:
This works fine, just one minor improvement I would suggest, in case user does not specify destination we can generate code in user home and at the end of execution we display where is code generated. This way it should be only `mvn spring-boot:run` for user, not confusion/complexity at all. Destination works well for advance users.

Reply:
I see it works well with men spring-boot, exactly what I was expecting my comment above, but why are we using spring-boot here? I think here we are using spring boot just for packaging and running code, right?

Reply:
Yes, spring-boot:run will be only used by the developer of MSG. The user will normally run with java -jar.

Reply:
I see what you are saying, I think this is good for now.
MSG has spring-boot dependency because we need to use spring boot annotations while generating the code. MSG is not a spring boot application, it is a standalone JVM application. 
`spring-boot:run` is typically used to start a spring boot microservice, while we are using it to run a standalone JVM application, I am not sure if it is a good idea or not, it needs more thinking but for now we can use this. Created issue https://github.com/Vipin-Sharma/MSG/issues/21 for this.